README.md

```diff
@@ -1,4 +1,39 @@
 # Keyword Search on KBFS Data
 
-Supports keyword search on data stored in KBFS.
+Supports keyword search on encrypted data stored in KBFS.
 
+This is an early prototype simulating the whole process locally with clients and
+a server. The network traffic is simulated by applying latency and bandwidth to
+the time logging.
+
+To run the demo, simply do:
+
+```
+go run main.go
+```
+
+Or better, with time logging:
+
+```
+go run main.go --enable_logger
+```
+
+The clients have a defualt mount point of `.client_fs/` and the server has a
```

Comment:
Also, "mount point" means something specific in reference to a file system.  I think here you just mean a directory for state storage?